### PR TITLE
add test config file

### PR DESCRIPTION
**Pull request by tbro**
Move test options to profiles. This helps clean up the command line a little, and should add more flexibility for future changes. Also addes `--no-fail-fast` option to run all the tests even after one fails.

### CODE REVIEW

File: .github/workflows/test-demo-native-marketplace.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ on:
       - "20[0-9][0-9][0-1][0-9][0-3][0-9]*"
   schedule:
     - cron: "0 0 * * 1"
-  pull_request:
```
Comment (imabdulbasit) [inline .github/workflows/test-demo-native-marketplace.yml:13]: why this change?

Comment (tbro) [reply .github/workflows/test-demo-native-marketplace.yml:13]: The thought was that we can just run marketplace test manually for now.

Comment (tbro) [reply .github/workflows/test-demo-native-marketplace.yml]: reverted 4e9f7428f7b2a5d7190bcd64b73ce301f7c77496

**Review by imabdulbasit — APPROVED** [review]
